ninja Archetype IT 'basic' failed: Cannot run additions goals in 6.4.2

maven(version 3.5.4) build command is `mvn clean install -DskipTests` using jdk1.8 under windows OS
and result in 
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:2.2:integration-test (default-integration-test) on project ninja-servlet-jpa-blog-archetype:
[ERROR] **Archetype IT 'basic' failed: Cannot run additions goals.**
[ERROR] -> [Help 1]
```
inspite of exclude archetype modules,how can I successfully build source ?

--- a/ninja-core/src/site/markdown/developer/changelog.md
+++ b/ninja-core/src/site/markdown/developer/changelog.md
@@ -1,6 +1,7 @@
 Version 6.8.2
 =============
 
+* 2022-01-10 Updated Maven Archetype plugin to 3.2.1 (thibaultmeyer)
 * 2021-12-18 Added Jackson Module to handle Java8 and Joda data types (thibaultmeyer)
 * 2021-12-24 Fixed NullPointerException on "Message::getWithDefault" when value and default value are both null
   (thibaultmeyer)

--- a/pom.xml
+++ b/pom.xml
@@ -222,7 +222,7 @@
 
                 <plugin>
                     <artifactId>maven-archetype-plugin</artifactId>
-                    <version>3.0.1</version>
+                    <version>3.2.1</version>
                 </plugin>
                 
                 <plugin>

